backend/fixtures-generator/fixtures_faker/providers/geo/__init__.py

```diff
@@ -4,7 +4,7 @@
 class Provider(GeoProvider):
 
     def local_latitude(self):
-        return self.coordinate(center=46.7716, radius=0.5)
+        return self.coordinate(center=46.7716, radius=0.05)
 
     def local_longitude(self):
-        return self.coordinate(center=7.0382, radius=0.25)
+        return self.coordinate(center=7.0382, radius=0.05)
```

Comment:
With those coordinates, the shops are closer together, which makes it nicer to see the fixture data on the map.